cluster/restore-from-backup.sh

```diff
@@ -101,7 +104,7 @@ wait_for_cluster_healthy() {
 # Wait until etcd and apiserver pods are down.
 wait_for_etcd_and_apiserver_down() {
   for i in $(seq 120); do
-    etcd=$(docker ps | grep etcd | grep -v etcd-empty-dir | grep -v etcd-monitor | wc -l)
+    etcd=$(docker ps | grep -e etcd-server -e etcd-server-events | wc -l)
```

Comment:
Is the "-e etcd-server-events" needed?

I think that "-e etcd-server" already matches "etcd-server-events", right?

@roberthbailey - this naming is consistent with what is in cluster/saltbase/salt/manifest, so I don't think it's GKE/GKE-specific now.

Reply:
Hmm.. Good point -- in fact etcd-server-events is not needed.